post-result.py.in

```diff
@@ -176,16 +181,17 @@ def post_comments_to_github(comments, github_pr_url, headers):
         return
                 
 def main():
-    OUTPUT = '```\n'
+    OUTPUT = ""
     job_name = BUILD_URL.split('/')[-3]
     build_number = BUILD_URL.split('/')[-2]
-    
+
+    public_build_url = BUILD_URL.replace(JENKINS_URL, PUBLIC_JENKINS_URL)
     try:
         build_data = get_build_data(BUILD_URL)
         if build_data:
             build_name = build_data['fullDisplayName']
             build_result = build_data['result']
-            OUTPUT +=  "BUILD " + build_name + " : " + build_result + "\n"
+            OUTPUT +=  "BUILD [" + build_name + "](" + public_build_url + ") : " + build_result + "\n"
 
             sub_build_outputs = get_sub_builds(BUILD_URL, 1)
```

Comment:
A tip that I don't know whether reasonable:

```
def get_sub_builds(builds_url, depth=1)
#when invoked here, just 
get_sub_builds(BUILD_URL)
```
get_sub_builds(BUILD_URL) is clear than get_sub_builds(BUILD_URL, 1)